packages/plugingen/userop.json

```diff
@@ -0,0 +1,37 @@
+{
+  "chain": "arbitrum-sepolia",
+
+  "connection_config": {
+    "bundler": {
+      "rpcUrl": "https://arb-sepolia.g.alchemypreview.com/v2/fGnvtpbRdda2Tk4ewN5nWxoX0dscaV_4"
+    },
+    "public": {
+      "rpcUrl": "https://arb-sepolia.g.alchemy.com/v2/MlNZ2V-DTCdvd3jqSAfaf3yDMIBciA8F"
+    }
+  },
+
+  "entrypoint": "0.6.0",
+
+  "account": {
+    "owner": "cook skin carbon unfold bundle half garbage scheme utility zero start culture argue armed clean",
```

Comment:
Probably shouldn't have leaked this...

Reply:
sorry. removed now, but probably should use another mnemonic.